Yoast/developer
# Updates the REST API documentation

Fixes #321 

## Summary
This PR updates the REST API documentation: fixes some references in the plugin, introduces some minor textual improvements, and updates the REST API output to what is currently for the given URL. 

## Relevant technical choices
Technical choices that affect more than this issue:
- None.

## Test instructions
This PR can be acceptance tested by following these steps:
1. Open up the REST API documentation on the `customization/apis/rest-api/` route.
2. Confirm the documentation is correct and shows an updated example at the end. 

## Quality assurance
